frontends/verilog/verilog_parser.y

```diff
@@ -883,7 +883,7 @@ param_signed:
 param_integer:
 	TOK_INTEGER {
 		if (astbuf1->children.size() != 1)
-			frontend_verilog_yyerror("Syntax error.");
+			frontend_verilog_yyerror("Internal error in param_integer - should not happen?");
```

Comment:
I was not able to trigger this error as well, and I believe it should not happen.
I believe it was created as a copy of the equivalent error message on line 886.
As much as these look symmetrical, they are NOT, due to that param_integer is specified before param_real in the param_decl and localparam_decl rules.
As a result, TOK_INTEGER will be parsed *after* TOK_REAL, preventing the condition on line 885.